ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/image/ImageComponentView.kt

```diff
@@ -291,10 +340,30 @@ private fun previewImageComponentStyle(
     size: Size,
     fitMode: FitMode,
     overlay: ColorScheme? = null,
+    paddingValues: PaddingValues = PaddingValues(0.dp),
+    marginValues: PaddingValues = PaddingValues(0.dp),
+    border: Border? = Border(
+        width = 2.0,
+        color = ColorScheme(
+            light = ColorInfo.Hex(
+                androidx.compose.ui.graphics.Color.Cyan.toArgb(),
```

Comment:
We have this aliased like so:
```suggestion
                ComposeColor.Cyan.toArgb(),
```